src/styles/index.scss

```diff
@@ -1,3 +1,66 @@
+@import './utils/variable';
+@import './utils/mixin';
+
 body {
   margin: 0;
+  height: 100vh;
+
+  @include centered-content;
```

Comment:
The mixin 'centered-content' is used here. Make sure it is defined in the './utils/mixin' file or elsewhere in your project.